src/interface/src/styleguide/treatment-expander/treatment-expander.component.ts

```diff
@@ -38,28 +37,31 @@ export class TreatmentExpanderComponent {
   /**
    * A treatment type
    */
-  @Input() treatmentType: 'SINGLE' | 'SEQUENCE' | null = null;
+  @Input() treatmentType: string | null = null;
```

Comment:
why `string` and not 'SINGLE' | 'SEQUENCE' , or even better `keyof typeof PRESCRIPTIONS`?